dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -5,7 +5,7 @@
 
 
 # Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
-export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.05.11T0411'
```

Comment:
This script isn't used at all by Jenkins actually.  Might end up removing it from the repo once we prove out the pipeline.